static/js/import_wizard2/components/mapping_page.tsx

```diff
@@ -55,9 +56,32 @@ export function MappingPage(props: MappingPageProps): JSX.Element {
   }
 
   useEffect(() => {
-    // TODO(beets): Use server-side detection API.
-    const predictedMapping = getPredictions(props.csvData, placeDetector);
-    setPredictedMapping(predictedMapping);
+    /* Build JSON of column data for detection API Request */
+    const column_ids = new Map(
+      props.csvData.orderedColumns.map((column) => {
+        return [column.columnIdx, column.id];
+      })
+    );
+    const column_headers = new Map(
+      props.csvData.orderedColumns.map((column) => {
+        return [column.columnIdx, column.header];
+      })
+    );
+    const columnParameters = new Map<string, unknown>([
```

Comment:
usually would prefer not to use the "unknown" type

Reply:
Switching to object fixed this!